apps/frontend/src/features/ai-ask/components/ChatInputFooter.tsx

```
@@ -0,0 +1,156 @@
+  maxQuestionLength,
+}: ChatInputFooterProps) => {
+  const theme = useTheme();
+
+  const remainingChars = maxQuestionLength - input.length;
+  const isDisabled = isStreaming || input.trim().length === 0;
+
+  return (
+    <footer css={footerStyle(theme)} aria-label="질문 입력">
+      <form css={inputBarStyle} onSubmit={onSubmit} aria-label="AI 질문 입력 폼" noValidate>
+        <label htmlFor="ai-question-input" css={srOnlyStyle}>
+          AI에게 질문하기
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)